### DISCUSSION TITLE
Beta-Release of Version 1.8.0

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#297] [Team Posts]
Dear all,

I released version 1.8.0 of our app today, but at the moment only as a public beta -- there is a lot of new technology in there that needs to be tested (and a few translations are also missing). If you have the time, I would appreciate if you could go to Google Play

https://play.google.com/store/apps/details?id=de.akaflieg_freiburg.enroute

There, you can enroll yourself in the beta-testing program and then (after a few minutes waiting time) download and the new app to your device(s). I would love to hear your feedback!

Best wishes -- stay healthy,

Stefan.

**Comment by HeinerTholen** [comment]
Here's the google guide on how to add yourself to the beta program: 
https://support.google.com/googleplay/answer/7003180?hl=en

**Comment by acrovato** [comment]
Hi there,

I went through the different menus while on the ground and everything looks fine. French translations are also OK and text is displaying correctly, at least on my phone. Additionally, I went flying today and I could check the app in-flight. I was focused on flying so I could not check everything thoroughly, but the app behaves as expected.

In brief, so far so good!
